remoulade/api/main.py

```diff
@@ -59,6 +59,12 @@ def enqueue_message():
     return {"result": "ok"}
 
 
+@app.route("/declared_actors")
+def get_declared_actors():
+    actors = get_broker().get_declared_actors()
```

Comment:
let's do a bit more than than than. Get the actors and return name, queue_name, and priority (define a to_dict method in actor). We cannot really serialize options as we don't know if it's serializable (maybe in the future).